content/docs/fundamentals/converting/resolving-errors@id.md

```diff
@@ -0,0 +1,350 @@
+---
+$title: Mengatasi error validasi
+$order: 2
+toc: true
+---
+
+[TOC]
+
+Pada bagian ini, kita akan membahas dan menyelesaikan error validasi AMP dari halaman AMP.  Perhatikan bahwa error mungkin muncul dalam urutan yang berbeda di konsol Anda.
+
+## Sertakan charset
+
+Kita akan memulai dengan memperbaiki error berikut:
+
+<pre class="error-text">
+Tag wajib 'meta charset=utf-8' tidak ada atau salah.
+</pre>
+
+Untuk menampilkan teks dengan benar, AMP mewajibkan Anda menentukan charset untuk halaman. Informasi charset meta juga harus menjadi turunan pertama tag `<head>`. Tag ini harus menjadi turunan pertama untuk menghindari konten yang ditafsirkan ulang ditambahkan sebelum tag charset meta.
+
+*Tambahkan** kode berikut sebagai baris pertama tag `<head>`:
+
+
+```html
+<meta charset="utf-8" />
+```
+
+**Simpan** file dan muat ulang halaman. Verifikasi bahwa error charset sudah tidak muncul.
+
+## Sertakan link kanonis
+
+Sekarang, mari kita lihat error berikut:
+
+<pre class="error-text">
+Tag wajib 'link rel=canonical' tidak ada atau salah.
+</pre>
+
+Setiap dokumen AMP harus memiliki link yang merujuk pada versi "kanonis" dokumen tersebut.  Kita akan mempelajari lebih lanjut apa itu halaman kanonis dan berbagai pendekatan terhadap penautan kanonis dalam langkah [Membuat halaman Anda dapat ditemukan](/id/docs/fundamentals/converting/discoverable.html) di tutorial ini.
+
+Untuk tutorial ini, kita akan menggunakan artikel HTML asli yang dikonversi menjadi halaman kanonis.
+
+Lanjutkan dan **tambahkan** kode berikut di bawah tag `<meta charset="utf-8" />`:
+
+```html
+<link rel="canonical" href="/article.html">
+```
+
+{% call callout('Catatan', type='note') %}
+Anda dapat membuat halaman AMP kanonis mandiri. Link kanonis masih diperlukan, tetapi harus mengarah ke artikel AMP itu sendiri:
+
+```html
+<link rel="canonical" href="article.amp.html">
+```
+{% endcall %}
+
+Sekarang, **muat ulang** halaman. Meskipun masih ada banyak error yang perlu diperbaiki, error pada link kanonis sudah tidak ada.
+
+## Tentukan atribut AMP
+
+AMP memerlukan atribut di elemen `<html>` utama halaman untuk menyatakan halaman sebagai dokumen AMP.
+
+<pre class="error-text">
+Atribut wajib '⚡' tidak ada di tag 'html ⚡ for top-level html'
+Tag wajib 'html ⚡ for top-level html' tidak ada atau salah.
+</pre>
+
+Error di atas dapat diatasi hanya dengan menambahkan atribut `⚡` ke tag `<html>` seperti berikut:
+
+```html
+<html ⚡ lang="en">
+```
+
+Sekarang, lanjutkan, muat ulang halaman, dan pastikan kedua error sudah tidak ada.
+
+{% call callout('Catatan', type='note') %}
+Meskipun direkomendasikan untuk menentukan `⚡`, kita juga dapat menggunakan atribut `amp` untuk menggantikan atribut `⚡`, seperti yang berikut:
+
+```html
+<html amp lang="en">
+```
+{% endcall %}
+
+## Tentukan viewport
+
+Selanjutnya, mari kita selesaikan error berikut:
+
+<pre class="error-text">
+Tag wajib 'meta name=viewport' tidak ada atau salah.
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).

Reply:
Updated code block with EN version.